changelog/1263.doc.2.rst

```diff
@@ -0,0 +1,2 @@
+Updated the documentation guide within the contributor guide to describe
+how to add references in BibTeX format.
```

Comment:
```suggestion
Updated the documentation guide to include a description on how to
add and cite references to PlasmaPy's global bibliography BibTeX file,
``bibliography.bib``.
```

Your write up did a little more than what your change log entry suggested, so I think this better encompasses the work.